WebAPI/Properties/launchSettings.json

```diff
@@ -14,7 +14,7 @@
       "dotnetRunMessages": true,
       "launchBrowser": true,
       "launchUrl": "swagger",
-      "applicationUrl": "https://localhost:7220;http://localhost:5180",
+      "applicationUrl": "http://localhost:5180;https://localhost:7220",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
Keep in mind that setting environment here to Development will look for appsettings.development.json